tests/testsuite/build.rs

```diff
@@ -1230,6 +1230,7 @@ fn crate_env_vars() {
         homepage = "https://example.com"
         repository = "https://example.com/repo.git"
         authors = ["wycats@example.com"]
+        license = "MIT"
```

Comment:
This is a nit, but if you're going to give an example license, please use the standard Rust dual-license, `MIT OR Apache-2.0`.

Reply:
I just thought it would be a nice example.
I choosed MIT, because it's in the `SPDX license list`.
See https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields and https://spdx.org/licenses/

@alexcrichton should I change it to `MIT OR Apache-2.0`, or is it okay for the testsuite? :)

Reply:
People often copy examples, and this issue is likely to be linked to from release notes or similar.

Reply:
I personally don't have an opinion on this, and I think following @joshtriplett's suggestion is fine.

Reply:
@joshtriplett Should be done, pipeline is running through. :)